src/resources/elements/ckEditor/pckeditor-text.ts

```diff
@@ -45,11 +63,18 @@ export class PCkeditorText {
           editor.editing.view.document.on("clipboardInput", (evt, data) => {
             const dataTransfer = data.dataTransfer;
             const textContent = dataTransfer.getData("text/plain");
-
+            const totalDataLength = textContent.length + editor.getData().length;
             if (!textContent) {
               return;
             }
             const viewContent = marked(textContent);
+            if (totalDataLength >= 500) {
+              const limit = editor.getData().startsWith("<p>") && editor.getData().endsWith("</p>") ? 507 : 500;
```

Comment:
Can you define `507` as a constant?
The `500` also please (for 500 the other places in the code as well)

Reply:
Would be nice to have 507 be dependent on 500, eg.
507 = 500 + otherValue